RSSReader/View/FeedCell.h

```diff
@@ -7,16 +7,20 @@
 
 #import <UIKit/UIKit.h>
 #import "FeedItem.h"
+#import "CustomTableViewProtocol.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface FeedCell : UITableViewCell
```

Comment:
![image](https://user-images.githubusercontent.com/48133328/104762079-d8335e00-5774-11eb-9175-7334a9f43223.png)
Some titles did not shown correctly.

Reply:
Full title of third cell is 
```objc
@"NIVEA отказалась спонсировать чемпионат мира по хоккею, если он пройдет в Беларуси"
```

Reply:
fixed